src/gui_common/fonts/Lato-Regular-SlightlySmall.tres

```diff
@@ -0,0 +1,18 @@
+[gd_resource type="DynamicFont" load_steps=7 format=2]
+
+[ext_resource path="res://assets/fonts/Lato-Regular.ttf" type="DynamicFontData" id=1]
+[ext_resource path="res://assets/fonts/NotoSans-Regular.ttf" type="DynamicFontData" id=2]
+[ext_resource path="res://assets/fonts/NotoSansHebrew-Regular.ttf" type="DynamicFontData" id=3]
+[ext_resource path="res://assets/fonts/NotoSansCJKsc-Regular.otf" type="DynamicFontData" id=4]
+[ext_resource path="res://assets/fonts/NotoSansCJKtc-Regular.otf" type="DynamicFontData" id=5]
+[ext_resource path="res://assets/fonts/NotoEmoji-Regular.ttf" type="DynamicFontData" id=6]
+
+[resource]
+size = 18
```

Comment:
This file should be named `AlmostSmaller` to be consistent with the existing files (`Jura-DemiBold-AlmostSmaller.tres`)